src/main/resources/reference.conf

```diff
@@ -4,15 +4,15 @@
 
 kamon {
   spm {
-    receiver-url = "https://spm-receiver.sematext.com/receiver/v1/_bulk"
+    receiver-url = "https://spm-receiver.sematext.com/write?db=metrics"
```

Comment:
Do we explain in UI instructions how to change this url when EU region or on-premises setup is used?

Reply:
Yes. EU instructions have instructions to change metrics & trace receiver URLs

Reply:
Those instructions need adjustments as part of this change. They will be a separate PR.